modules/serverless-broker-types.adoc

```diff
@@ -1,27 +1,21 @@
 // Module included in the following assemblies:
 //
-// * /serverless/develop/serverless-using-brokers.adoc
+// * /serverless/discover/serverless-brokers.adoc
 
 :_content-type: CONCEPT
 [id="serverless-broker-types_{context}"]
 = Broker types
 
-There are multiple broker implementations available for use with {ServerlessProductName}, each of which have different event delivery guarantees and use different underlying technologies. You can choose the broker implementation when creating a broker by specifying a broker class, otherwise the default broker class is used. The default broker class can be configured by cluster administrators.
-// TO DO: Need to add docs about setting default broker class.
+You can choose the broker implementation when creating a broker by specifying a broker class, otherwise the default broker class is used. The default broker class can be configured by cluster administrators.
 
-[id="serverless-broker-types-channel-based"]
-== Channel-based broker
+[id="serverless-broker-types-default_{context}"]
+== Default broker implementation for development purposes
 
-The channel-based broker implementation internally uses channels for event delivery. Channel-based brokers provide different event delivery guarantees based on the channel implementation a broker instance uses, for example:
+Knative provides a default channel-based broker implementation that is useful for development and testing purposes. This default broker does not provide adequate event delivery guarantees for production environments, and is backed by the `InMemoryChannel` channel implementation by default.
```

Comment:
`that is useful for development` -> I'd not even say that.

I'd say: In combination with the InMemoryChannel it is useful for development.

But configuring the default broker w/ a kafkachannel is also not really great idea, bc of the extra network hops it produces

Reply:
>But configuring the default broker w/ a kafkachannel is also not really great idea, bc of the extra network hops it produces

We mention all of that already though AFAIK, i.e. that we don't recommend doing this, we recommend using the KafkaBroker instead and that one of the benefits of Kafka broker is fewer network hops.

Reply:
If you're not already, please use the preview links for the docs, since it's easier to see the full context of the docs

Reply:
Changed from "is useful for" to "you can use for" -  I think we need to at least say this or else why even document it? I understand we'd prefer that they use Kafka, but if this is still an option we should explain what it's for.